samples/nextjs-template/src/app/json/[...slug]/page.tsx

```diff
@@ -7,6 +11,19 @@ type Props = {
   }>;
 };
 
+function handleGraphErrors(err: unknown): never {
+  if (err instanceof GraphErrors.GraphResponseError) {
+    console.log('Error message:', err.message);
+    console.log('Query:', err.request.query);
+    console.log('Variables:', err.request.variables);
+  }
+  if (err instanceof GraphErrors.GraphContentResponseError) {
+    console.log('Detailed errors: ', err.errors);
+  }
```

Comment:
Why not `console.error()`?

Reply:
I assume this is only displayed in dev mode? Since this is just in the template, it doesn't really matter. But in production I assume one wants to log this is a proper error.